components/camel-smpp/src/main/java/org/apache/camel/component/smpp/SmppConfiguration.java

```diff
@@ -733,6 +735,26 @@ public void setSingleDLR(boolean singleDLR) {
         this.singleDLR = singleDLR;
     }
 
+    public String getInterfaceVersion() {
+        return interfaceVersion;
+    }
+
+    /**
+     * Defines the interface version to be used in the binding request with the SMSC. The following values are allowed,
+     * as defined in the SMPP protocol:
+     * <ul>
```

Comment:
Since this will be used to generate documentation I would avoid ul and li, because for the Javadoc they will be good, but for generating the website they could be a mess.

Reply:
I have used the syntax that a lot of the other configuration fields have used (look at this file and scroll up for a few cases). I can see though that they are not really useful in the docs (https://camel.apache.org/components/4.0.x/smpp-component.html). Will remove.

Reply:
Looks good then. I never noticed that, maybe because I didn't work on this specific component. All good.